op-node/flags/flags.go

```diff
@@ -34,6 +34,18 @@ var (
 		Usage:   "Address of L2 Engine JSON-RPC endpoints to use (engine and eth namespace required)",
 		EnvVars: prefixEnvVars("L2_ENGINE_RPC"),
 	}
+	L2RpcTimeout = &cli.DurationFlag{
+		Name:    "l2.rpc-timeout",
+		Usage:   "Timeout for L2 RPC requests",
+		EnvVars: prefixEnvVars("L2_RPC_TIMEOUT"),
+		Value:   time.Second * 10,
+	}
+	L2RpcBatchTimeout = &cli.DurationFlag{
+		Name:    "l2.rpc-batch-timeout",
+		Usage:   "Timeout for L2 RPC batch requests",
```

Comment:
Why does it need to be 2 different flags? A batch request is still a request right?

Reply:
In the referenced section of the code, https://github.com/ethereum-optimism/optimism/blob/3b28485044c183e56a7cfe0b535eb8b247f7fbc7/op-service/client/rpc.go#L166-L176
the timeout configuration for batch RPC calls is set to differ from that of single RPC calls.

Reply:
I would be supportive of this if we used 1 flag and passed the same value to both

Reply:
Ok! I will use one value.